pinc/Project.inc

```diff
@@ -252,13 +257,14 @@ class Project
                 $charsuite->title
             ));
         }
-
+        validate_projectID($this->projectid);
```

Comment:
You don't need this because `$this->projectid` is not being used as a table, but as a field you are already escaping.